app/test/app.test.js

```diff
@@ -0,0 +1,60 @@
+const postModel = require('../src/model/post');
+test('check', async () => {
```

Comment:
niech kod będzie jakoś logicznie połączony/rozdzielony nowymi wierszami, fatalnie się to czyta ;p